assets/js/sidebar/sidebar-version-select.js

```diff
@@ -17,14 +18,38 @@ export function initialize () {
     const currentVersion = versionsContainer.textContent.trim()
 
     const latestVersionNode = versionNodes.find(node => node.latest)
-    const latestVersion = latestVersionNode?.version !== currentVersion ? latestVersionNode?.version : null
+    const latestVersion = latestVersionNode?.version !== currentVersion ? latestVersionNode?.url : null
 
     const nodes = decorateVersionNodes(versionNodes, currentVersion)
 
     renderVersionsDropdown({ nodes, latestVersion })
+    initializeVersionSelect()
   }
 }
 
+function initializeVersionSelect () {
+  const select = qs(VERSIONS_DROPDOWN_SELECTOR)
+  const text = qs(VERSIONS_TEXT_SELECTOR)
+
+  text.textContent = select.options[select.selectedIndex].text
+
+  text.addEventListener('click', (event) => {
```

Comment:
Did you find this simpler than setting up the width? My biggest concern is if this will work from mobile, I will check soon and let you know.